drivers/input/keyboard/brain-kbd-i2c.c

```diff
@@ -101,38 +143,34 @@ static irqreturn_t bk_i2c_irq_handler(int irq, void *devid)
 	n = raw >> 8;
 	k1 = raw & 0xff;
 
+	dev_dbg(&kbd->cli->dev, "kbd-raw0[%02X,%02X]\n",n,k1);
```

Comment:
すいません、レビューに入れ忘れてました。
デバッグ用メッセージはより説明的であるほうが望ましいです。たとえば:

```suggestion
	dev_dbg(&kbd->cli->dev, "N=%d, Raw key event: %02X\n", n, k1);
```